src/payment/requestor.rs

```diff
@@ -116,8 +121,15 @@ impl PaymentRequestorApi {
         self.client.get(&url).send().json().await.or_else(default_on_timeout)
     }
 
-    pub async fn get_invoices(&self) -> Result<Vec<Invoice>> {
-        self.client.get("requestor/invoices").send().json().await
+    pub async fn get_invoices<Tz>(&self, after_timestamp: Option<DateTime<Tz>>, max_items: Option<i64>) -> Result<Vec<Invoice>> where Tz: TimeZone, Tz::Offset: Display, {
```

Comment:
I'd rather go with `u64` for `max_items` since negative values do not make sense here.

Reply:
replied above, the descision is between casting and following diesel..
in case of casting i propose a `u32`

Reply:
There's nothing wrong with casting. Let's do `u32`.

Reply:
unwraps are wrong with casting :P ( but not for a safe u32 -> i64 one )